src/models/stackedAreaChart.js

```diff
@@ -12,7 +12,6 @@ nv.models.stackedAreaChart = function() {
         , legend = nv.models.legend()
         , controls = nv.models.legend()
         , interactiveLayer = nv.interactiveGuideline()
-        , tooltip = nv.models.tooltip()
```

Comment:
Can we remove tooltip altogether? Or is it being kept for legacy or other reasons?